VideoApi/VideoApi.Domain/Conference.cs

```diff
@@ -223,8 +223,12 @@ public void AddLinkedParticipants(Guid primaryParticipantRefId, Guid secondaryPa
             var secondaryParticipant =
                 Participants.Single(x => x.ParticipantRefId == secondaryParticipantRefId);
 
-            primaryParticipant.AddLink(secondaryParticipant.Id, linkedParticipantType);
-            secondaryParticipant.AddLink(primaryParticipant.Id, linkedParticipantType);
+            if (primaryParticipant is Participant && secondaryParticipant is Participant)
```

Comment:
Not major but could we change this to the pattern seen elsewhere?
`primaryParticipant is Participant primaryParticipantCasted && secondaryParticipant is Participant secondaryParticipantCasted`
Or would that not work as we're calling AddLink method?